packages/jest-matchers/src/__tests__/toThrowMatchers-test.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+jsinfra
+ */
+
+'use strict';
+
+describe('.toThrowError()', () => {
+  describe('strings', () => {
+    it('passes', () => {
+      expect(() => { throw new Error('lol'); }).toThrowError('lol');
```

Comment:
I think `lol` and `lmao` are pretty awkward in tests. Can we fall back to `banana` and `apple` like `jest-haste-map` does? :)